### PR TITLE
Added support for bower package manager

### PR DESCRIPTION
**Pull request by brunowego**
Please make a tag with correct version like `0.0.1` and create a master branch.

After this, users can install this package using bower:

``` bash
bower install awesomplete
```

### CODE REVIEW

**Comment by aboutaaron** [comment]
:+1:

**Comment by LeaVerou** [comment]
Thanks!
Re:master branch, users can already install bower, they just have to add #gh-pages to the URL.

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "awesomplete",
+  "main": [
+    "./awesomplete.css",
+    "./awesomplete.js"
+  ],
+  "version": "0.0.0",
+  "homepage": "https://github.com/LeaVerou/awesomplete",
+  "authors": [
+    "Lea Verou <>"
```
Comment (arthurvr) [inline bower.json:10]: @LeaVerou You will have to actually fill this out too :)

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  ],
+  "description": "Ultra lightweight, usable autocomplete with zero dependencies.",
+  "keywords": [
+    "autocomplete",
+    "lightweight"
```
Comment (arthurvr) [inline bower.json:15]: Can we find more `keywords`? 2 is pretty minimal.

**Comment by arthurvr** [comment]
@LeaVerou These days it's best practice to submit front-end stuff to NPM too. PR with a `package.json` welcome?

**Comment by brunowego** [comment]
When try register in bower repository I get this message:

``` bash
[~] bower register awesomplete git@github.com:LeaVerou/awesomplete.git                                                                                                                            19:48:46
bower                          convert Converted git@github.com:LeaVerou/awesomplete.git to git://github.com/LeaVerou/awesomplete.git
bower awesomplete#*            resolve git://github.com/LeaVerou/awesomplete.git#*
bower awesomplete#*       ENORESTARGET Tag/branch master does not exist

Additional error details:
No tags found in git://github.com/LeaVerou/awesomplete.git
Available branches: gh-pages
```

After register the users can search your package.

File: bower.json (hunk 1 above)
Comment (LeaVerou) [inline bower.json:10]: With what? Email?

**Comment by LeaVerou** [comment]
You need to specify the branch too, because it doesn’t have a master branch. See https://github.com/LeaVerou/prism/issues/180

**Comment by LeaVerou** [comment]
@arthurvr Sure, thanks!

**Comment by rbecheras** [comment]
The right way is to specify a version number in bower.json and create a matching git tag

**Pull request merged**